sqlalchemy_aio/engine.py

```diff
@@ -293,12 +303,31 @@ def __init__(self, result_proxy, run_in_thread):
         self._result_proxy = result_proxy
         self._run_in_thread = run_in_thread
 
+    async def __aiter__(self):
+        """Like :meth:`ResultProxy.__iter__\
+        <sqlalchemy.engine.ResultProxy.__iter__>`, but is a coroutine.
+        """
+        iterator = iter(self._result_proxy)
+        while True:
+            try:
+                row = await self._run_in_thread(iterator.__next__)
+            except StopIteration:
+                raise StopAsyncIteration
+            else:
+                yield row
```

Comment:
This should probably be `return`. Ref: https://travis-ci.org/RazerM/sqlalchemy_aio/jobs/260548712#L314